code/graph.q

```diff
@@ -70,21 +70,21 @@ graph:.ml.connectEdge[graph;`labelEncode        ;`target  ;`trainTestSplit;`targ
 graph:.ml.connectEdge[graph;`dataCheck          ;`config  ;`trainTestSplit;`config]
 
 // Select_Models
-graph:.ml.connectEdge[graph;`dataCheck      ;`config;`selectModels;`config]
```

Comment:
Need to update the graph in line with the below changes to outline logical requirement and new ordering